### PR TITLE
CRI-O: create /data/gcs

### PR DESCRIPTION
**Pull request by runcom**
@stevekuznetsov ptal

Signed-off-by: Antonio Murdaca <runcom@redhat.com>

### CODE REVIEW

File: sjb/config/common/test_cases/crio.yml [hunk 1]
```diff
@@ -82,6 +82,12 @@ actions:
         echo "Bad job type provided in JOB_SPEC: $job_type"
       fi
 post_actions:
+  - type: "script"
+    title: "craete and chwon /data/gcs for artifacts"
```
Comment (mrunalp) [inline sjb/config/common/test_cases/crio.yml]: nit: craete -> create

Comment (runcom) [reply sjb/config/common/test_cases/crio.yml]: fixed thanks!

**Comment by cevich** [comment]
@runcom LGTM FWIW (now that I understand it better) :smile:

**Pull request merged**